sdk/networkcloud/azure-resourcemanager-networkcloud/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0 (2023-08-17)
+## 1.0.0 (2023-08-25)
```

Comment:
Why we change the release date here?

Reply:
Their target is 8/25

I just being lazy so that if they release as expected, I don't need to change it again later.